### PR TITLE
New: Epic playtime import support

### PR DESCRIPTION
**Pull request by darklinkpower**
Adds playtime import to the Epic library plugin. Based on research done by joyrider3774.

I have verified that:
- [x] These changes work, by building the application and testing them.
- [x] Pull request is targeting `devel` branch.
- [x] I added myself into [contributors file](https://github.com/JosefNemec/Playnite/blob/devel/source/Playnite.DesktopApp/Resources/contributors.txt) if I want to receive contribution credit in the application itself.

### CODE REVIEW

File: source/Plugins/EpicLibrary/Services/EpicAccountClient.cs [hunk 1]
```diff
@@ -233,6 +247,11 @@ private void renewToknes(string refreshToken)
         {
             using (var httpClient = new HttpClient())
             {
+                if (url.Contains("/library/api/public/playtime/account/"))
```
Comment (JosefNemec) [inline source/Plugins/EpicLibrary/Services/EpicAccountClient.cs:250]: I don't think this should be placed here. Generic method like this should not be doing arbitrary modifications to passed parameters. Url modifications should be done in GetPlaytimeItems.

Comment (darklinkpower) [reply source/Plugins/EpicLibrary/Services/EpicAccountClient.cs:250]: I agree. I was not sure if this was a good idea for that reason but decided this initially to not have to load the tokens twice.

Comment (darklinkpower) [reply source/Plugins/EpicLibrary/Services/EpicAccountClient.cs]: Updated PR with the change

**Comment by JosefNemec** [comment]
Merged manually into new extensions repo https://github.com/JosefNemec/PlayniteExtensions/commit/9cbc78452cd4acd0515d459344e4bad587fe2772

**Pull request closed without merging**